lib/clearance/configuration.rb

```diff
@@ -121,12 +121,20 @@ def  user_actions
       end
     end
 
+    # The name of user parameter for the configured user model.
+    # This is derived from the `model_name` of the `user_model` setting.
+    # In the default configuration, this is `user`.
+    # @return [Symbol]
+    def user_parameter
+      user_model.model_name.singular.to_sym
```

Comment:
It was a pre-existing requirement (see `user_id_parameter`) and I've said many times this gem has a dependency on ActiveRecord and other ORMs are not supported. So I think we're good.